.circleci/config.yml

```diff
@@ -1,248 +1,131 @@
-          command: ./.ci/test/visual-regression/run
-
-      - store_artifacts:
-          path: /tmp/artifacts
-          destination: artifacts
-
-  composer_lock_updater:
-    <<: *defaults
-    steps:
-      - checkout
-
-      - run:
-          # Set TERMINUS_ENV and related environment variables.
-          # https://github.com/pantheon-systems/docker-build-tools-ci/blob/6.x/scripts/set-environment
-          name: setup-environment-vars
-          command: /build-tools-ci/scripts/set-environment
-
-      - run:
-          name: Authenticate with Terminus
-          command: terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
-
-      - run:
-          name: run composer lock updater
-          command: set -e && terminus project:clu $TERMINUS_SITE
-
-workflows:
+version: 2.1
+
+# https://app.circleci.com/pipelines/github/prodigyjoe/test-urban-circleci-site
+
+# ## General Environment Variables set in CircleCI
+
+# * TERMINUS_TOKEN created in https://dashboard.pantheon.io/users/#account/tokens/
+# * DEFAULT_BRANCH set to `main` to ensure our primary branch delivers to dev
+
+# ## Setup Details
+
+# * Review https://pantheon.io/docs/drupal-9#create-a-new-drupal-9-site-with-integrated-composer
+# * Create new site from Drupal 9 upstream: https://dashboard.pantheon.io/sites/create?upstream_id=e96c6794-77fe-4931-9a20-48a2fe1a3789
+# * Clone Pantheon locally, update pantheon.yml to include `build_step: false`
+# * Copy `pantheon.yml` and `pantheon.upstream.yml` into your own repo
+# * Ensure the above environment variables in CircleCI
+
+workflows: #Defines a list of jobs and their run order
   version: 2
-  build_deploy_and_test:
-    jobs:
-      - configure_env_vars
-      - static_tests
-      - deploy_to_pantheon:
-          requires:
-            - static_tests
-      - visual_regression_test:
-          requires:
-            - configure_env_vars
-            - deploy_to_pantheon
-          filters:
-            branches:
-              ignore:
-                - master
-      - behat_test:
-          requires:
-            - deploy_to_pantheon
-            - visual_regression_test
-
-  scheduled_update_check:
-    triggers:
-       - schedule:
-           cron: "0 0 * * *"
-           filters:
-             branches:
-              only:
-                - master
-    jobs:
-      - composer_lock_updater
+  build_and_push: #name of workflow(runs job named build and job named push)
+      jobs: #A collection of steps, which run commands/scripts as required. Must declare an executor that is either docker, machine, windows, or macos.
+        - build #Run job called build
+        - pantheon/push:  #Run pantheon orb's job named push
+            requires: #Require a dependency by job name
+              - build #Require a job named build
+            checkout: false #Set pantheon orb's push job's paramater checkout to false
+            terminus_clone_env: live  #Set pantheon orb's push job's paramater terminus_clone_env to live
```

Comment:
DOes your live site have drupal installed?

Reply:
Your dev environment has drupal installed, so you can change this to pull content from the `dev1 environment instead of Live.